android/app/src/main/java/org/smartregister/fhircore/viewmodel/PatientListViewModel.kt

```diff
@@ -129,6 +131,25 @@ class PatientListViewModel(application: Application, private val fhirEngine: Fhi
     viewModelScope.launch { fhirEngine.syncUpload() }
   }
 
+  fun isPatientExists(id: String, callback: OnPatientSearchResult) {
+    viewModelScope.launch {
+      try {
+         fhirEngine.load(Patient::class.java, id)
+         callback.onSearchDone(true, id)
+      } catch (e: ResourceNotFoundException) {
+         callback.onSearchDone(false, id)
+      }
+    }
+  }
+
+  fun clearPatientList() {
+    liveSearchedPaginatedPatients.value =
+            Pair(
+                    listOf(),
```

Comment:
The function `listOf()` the equivalent of `emptyList()` when no value is passed but `emptyList()` to me appears to communicate that intention of instantiating a `Nothing` list in Kotlin

Reply:
done